src/object/srv_obj.c

```diff
@@ -1609,9 +1663,17 @@ obj_local_rw_internal(crt_rpc_t *rpc, struct obj_io_context *ioc,
 			rc = rc1;
 			break;
 		}
+		D_INFO("CPU yield detected,  dtx_handle_resend() rc1=%d, rc=%d\n", rc1 ,rc);
```

Comment:
(style)  space prohibited before that ',' (ctx:WxV)
(style)  space required after that ',' (ctx:WxV)